app/uk/gov/hmrc/vulnerabilities/service/XrayService.scala

```diff
@@ -59,7 +64,7 @@ class XrayService @Inject()(
     }
   }
 
-  private def generateReport(svd: ServiceVersionDeployments)(implicit hc: HeaderCarrier): EitherT[Future, Status, Report] =
+  private def generateReport(svd: ServiceVersionDeployments)(implicit hc: HeaderCarrier): EitherT[Future, Status, (Int, Report)] =
```

Comment:
Is it worth adding the `reportId` to `Report` itself? Or is it of no value after the report has been deleted?
Might just require adding `reportId` to `Report.apiFormat`

Reply:
I dont think its used outside of deleting the report on fail and for the above case. Given we are removing the reports immediately after they have been set in the repo now it doesnt seem necessary to keep them.